typescript/packages/well-log-viewer/src/SyncLogViewer.tsx

```diff
@@ -404,9 +418,11 @@ class SyncLogViewer extends Component<SyncLogViewerProps, State> {
                 this.props.wellpickFlatting,
                 prevProps.wellpickFlatting
             ) ||
-            this.props.welllogs.length !== prevProps.welllogs.length
+            this.props.welllogs?.length !== prevProps.welllogs?.length ||
+            this.props.wellLogCollections?.length !==
```

Comment:
this.props.wellLogCollections?.length ?? this.props.welllogs?.length !== prevProps.wellLogCollections?.length ?? prevProps.welllogs?.length